src/create-mock-list.ts

```diff
@@ -1,2 +1,9 @@
 import { DeepPartial } from './partial/deepPartial';
-export declare function createMockList<T extends object>(quantity: number, iterator?: (index: number) => DeepPartial<T>): T[];
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function createMockList<T extends object>(quantity: number, iterator?: (index: number) => DeepPartial<T>): T[] {
+  throw new Error(`
```

Comment:
Can we try to extract the Error in one file? something like ts-auto-mock-error.ts.

Reply:
done, also updated error wording